packages/solutions/module-tools/src/builder/feature/redirect.ts

```diff
@@ -89,9 +92,31 @@ async function redirectImport(
         }
       }
 
+      if (redirect.autoExtension) {
+        if (ext === '') {
+          // add extension for relative path, no check if it's a directory.
+          if (jsExtension !== '.js' && name.startsWith('.')) {
+            str.overwrite(start, end, `${name}${jsExtension}`);
```

Comment:
Conflict with https://github.com/web-infra-dev/modern.js/blob/main/packages/review/eslint-config-app/base.js#L1084

Reply:
We can disable `import/no-useless-path-segments`, this rule is no longer recommended.

Reply:
I have removed it